example_project/requirements.txt

```diff
@@ -1,5 +1,5 @@
-Django~=2.2.9
-django-environ==0.4.5
-django-rosetta==0.9.3
+Django~=3.2.20
```

Comment:
should we =3.2.20,<6

Reply:
@cmaggiulli In general capping versions is often not recommended as it removes the end consumer from being able to, for example, use an older django gaurdian that works fine with new Django but has not yet been tested at the time.

I think the general consensus is that it's better to leave these open and let the end consumer pin upper bounds in applications if needed but leave libraries more permissive. Especially for a library like Django that has stable backwards compatibility story.

Reply:
Leaving it as is.

Reply:
`~=` already includes an upper bound.  Also: This an example project, not django-guardian's requirements.